nifi-framework-bundle/nifi-framework/nifi-headless-server/src/main/java/org/apache/nifi/headless/HeadlessNiFiServer.java

```diff
@@ -107,7 +110,7 @@ public void start() {
             logger.info("Loading Flow...");
 
             FlowFileEventRepository flowFileEventRepository = new RingBufferEventRepository(5);
-            AuditService auditService = new HeadlessAuditService();
+            AuditService auditService = new ReportableAuditService(new HeadlessAuditService(), new HeadlessFlowActionReporter(), new HeadlessActionConverter());
```

Comment:
This change is unnecessary, and actually indicates that the `Headless` implementation classes can be removed.